### PR TITLE
added a ubiquiti page

### PR DESCRIPTION
**Pull request by IronicBadger**

### CODE REVIEW

File: docs/system-administration/ubiquiti.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+This page contains information relating to the Ubiquiti line of products.
+
+## Adopt a new Access Point with remote Unifi controller software
+
+If your Unifi controller software doesn't run on your LAN it is neccessary to tell the AP where it is. This allows the AP to be 'adopted' by the controller and have its configuration managed by it.
+
+To do this you will need to SSH to the AP (default user and password is `ubnt`).
+
+    ssh <ip-of-access-point>
+    set-inform http://<controller-url>:8080/inform
```
Comment (RealOrangeOne) [inline docs/system-administration/ubiquiti.md]: Possibly tangential, super nitpick comment: Is the controller connection really over non-TLS?

File: docs/system-administration/ubiquiti.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Once you've done this, the AP should show up almost immediately in the interface of your Unifi controller saying 'pending adoption'.
+
+Adopt the AP and configuration for that site will be automatically applied.
+
+> Note that after adoption the SSH password will be changed from ubnt/ubnt. You can modify the post adoption Unifi SSH username and password in the controller software by enabling 'advanced features' and then configuring your desired credentials under `Site -> Device Authentication`.
```
Comment (RealOrangeOne) [inline docs/system-administration/ubiquiti.md]: I'd be interested in trying a proper note here

https://squidfunk.github.io/mkdocs-material/extensions/admonition/

File: docs/system-administration/ubiquiti.md (hunk 1 above)
Comment (IronicBadger) [reply docs/system-administration/ubiquiti.md]: yes, it is http. it's only the announce URL though, the controller itself is TLS via whatever means you pick. i could go into that but i'd say it's not really in scope for this article unless we write another and link to it. i wasn't too keen on that plan for now though #abitlazy

**Comment by IronicBadger** [comment]
> great to see it getting off the ground. happy to help if I can 😄
> 💯 🔥

feel free to submit a similar PR for something you're interested in elsewhere!

File: docs/system-administration/ubiquiti.md (hunk 2 above)
Comment (IronicBadger) [reply docs/system-administration/ubiquiti.md]: Will that work with hugo if / when we migrate? I've not come across this note thing before. I worry it's going to create technical debt?

**Comment by ndsamuelson** [comment]
> > great to see it getting off the ground. happy to help if I can 😄
> > 💯 🔥
> 
> feel free to submit a similar PR for something you're interested in elsewhere!

👍

File: docs/system-administration/ubiquiti.md (hunk 2 above)
Comment (BCNelson) [reply docs/system-administration/ubiquiti.md]: I don't believe that hugo will support this. Hugo generally uses short code to extend functionality. I'm not sure that that should stop us however If we want to really get a good idea of mkdocs then we should use it to its fullest.

Comment (RealOrangeOne) [reply docs/system-administration/ubiquiti.md]: These can be considered similar to Hugo's shortcodes. I think it's unrealistic to expect a port to Hugo be free in terms of content modifications, and as @BCNelson says, we shoud use `mkdocs` to it's fullest, as we aren't necessarily migrating.

**Comment by IronicBadger** [comment]
I finally got around to formatting the note. Please review.

**Pull request merged**